src/routes/chatbot/+page.svelte

```diff
@@ -0,0 +1,137 @@
+<script>
+	import { writable } from 'svelte/store';
+
+	let message = '';
+	let messages = writable([]);
+	let botResponses = [
+		'Hallo! Welkom bij de Openbare Bibliotheek Amsterdam. Waarmee kan ik u vandaag helpen?',
+		'Dat klinkt goed! Kunt u mij de titel of het onderwerp geven waarin u geïnteresseerd bent?',
+		'Ik kan daar zeker bij helpen! Zoekt u naar een specifiek boek of wilt u een lijst van aanbevelingen?',
+		'Dat boek is te vinden op de tweede verdieping, bij de sectie Literatuur. Wilt u nog iets anders weten?',
+		'Graag gedaan! Als u nog andere vragen heeft, aarzel niet om contact op te nemen. Fijne dag verder en we hopen u snel weer te zien in de OBA!'
+	];
+
+	function sendMessage() {
+		messages.update((currentMessages) => {
+			let updatedMessages = [...currentMessages, { text: message, sender: 'user' }];
+			message = '';
```

Comment:
Is het nodig om de message binding opnieuw een empty string value te geven? Dat deed je ook op regel 4

Reply:
Ja dit is nodig want de code op regel 4 zorgt ervoor dat de message in eerste instantie leeg is en de code die jij referenced zorgt ervoor dat het message veld weer leeg is na het verzenden van een message.